src/scripts/store/dimStoreItem.directive.js

```diff
@@ -91,6 +91,8 @@ function StoreItem(dimItemService, dimStoreService, ngDialog, dimLoadoutService,
     vm.clicked = function openPopup(item, e) {
       e.stopPropagation();
 
+      $rootScope.$broadcast('item-clicked', item);
```

Comment:
This seemed like the simplest way to get a promise loading in the background to me, so that DIM will be snappy even if the DTR API is sluggish.  It might have been a bad way to go!